Jenkinsfile

```
@@ -49,6 +49,7 @@ for (i = 0; i < buildTypes.size(); i++) {
                 '--update-snapshots',
                 "-Dmaven.repo.local=$m2repo",
                 '-Dmaven.test.failure.ignore',
+                '-DforkCount=0.5C',
```

or
```suggestion
                '-DforkCount=2',
```
to match the prior config of the `test` module for which performance is far more significant.

---

Perhaps that is also @basil’s point

> make test runs more deterministic